Lib/distutils/tests/test_check.py

```diff
@@ -99,6 +100,11 @@ def test_check_restructuredtext(self):
         cmd = self._run(metadata, strict=1, restructuredtext=1)
         self.assertEqual(cmd._warnings, 0)
 
+        # check that includes work to test #31292
+        metadata['long_description'] = f'title\n=====\n\n.. include:: {os.devnull}'
```

Comment:
Could you use explicit formatting, so that backporting the fix becomes easier?

Reply:
sure, will do in a second.